docs/maps/maps-getting-started.asciidoc

```diff
@@ -136,7 +138,7 @@ image::maps/images/grid_metrics_both.png[]
 . Set *Index pattern* to *kibana_sample_data_logs*.
 . Click the *Add layer* button.
 . Set *Layer name* to `Total Requests and Bytes`.
-. Set *Min zoom* to 0 and *Max zoom* to 9.
+. Set *Zoom range for layer visibility* to the range [0, 9].
```

Comment:
Where you show the map at the end of the page, instead of "Your map now looks like this" say "Your map now looks like this between between zoom levels x and x'>